static/DSP8010_1.0.0/json-schema-oem/RackHD.BootImage.json

```diff
@@ -62,6 +69,13 @@
                     "type": "string"
                 }
             },
+            "required": [
+                "domain",
+                "hostname",
+                "osName",
+                "rootPassword",
+                "dnsServers"
+            ],
```

Comment:
by RackHD design, all these fields are optional because all of them have a default value ( see https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/install-centos.js) , so we still want to enforce this required criteria in Redfish API?